### DISCUSSION TITLE
请问如何将应用下载链接由https://域名/download/releases/5 改成https://服务器ip/download/releases/5

### DISCUSSION THREAD

**Discussion by adaJQD** [tryzealot/zealot#1405] [Q&A] [answered]
默认生成的应用下载链接是基于域名的，例如 https://域名/download/releases/5
请问如何修改成直接是服务器ip地址的？例如 https://服务器ip/download/releases/5

**Comment by icyleaf** [comment, marked as answer]
在系统设置里把域名修改成 host:port 格式，需要重启服务器后生效
